cssjson.js

```diff
@@ -222,3 +222,5 @@ var CSSJSON = new function () {
     };
 
 };
+
+module.exports = CSSJSON;

```

Comment:
Added a new PR with just the bower.json. We're using webpack so not having the module.exports is not a problem in our case. Others would probably benefit from just the bower.json. Didn't have your email address so just put your web address in the author field.